card/mod/01_core/spec/set/all/fetch_spec.rb

```diff
@@ -10,196 +10,233 @@
-        #expect(Card.cache).to receive(:delete).with('y').exactly(2).times
+        # expect(Card.cache).to receive(:delete).with('x').exactly(2).times
+        # expect(Card.cache).to receive(:delete).with('y').exactly(2).times
 
         a.save!
       end
     end
+    describe 'default option' do
+      context "when card doesn't exist" do
+        it 'initializes new cards' do
+          card = Card.fetch 'non-existent',
+                            new: { default_content: 'default content' }
+          expect(card.content).to eq 'default content'
+        end
+      end
+      context 'when new card exist' do
+        it "doesn't change anything" do
+          Card.new name: 'new card',
+                   '+sub' => { content: 'some content' }
+          card = Card.fetch 'new card+sub',
+                            new: { default_content: 'new content' }
+          expect(card.content).to eq 'some content'
+        end
+      end
+    end
 
-    describe "preferences" do
+    describe 'preferences' do
       before do
         Card::Auth.current_id = Card::WagnBotID
       end
 
-      it "prefers db cards to pattern virtual cards" do
-        c1=Card.create!(name: "y+*right+*structure", content: "Formatted Content")
-        c2=Card.create!(name: "a+y", content: "DB Content")
-        card = Card.fetch("a+y")
+      it 'prefers db cards to pattern virtual cards' do
+        c1=Card.create!(name: 'y+*right+*structure', content: 'Formatted Content')
+        c2=Card.create!(name: 'a+y', content: 'DB Content')
+        card = Card.fetch('a+y')
         expect(card.virtual?).to be_falsey
-        expect(card.rule(:structure)).to eq("Formatted Content")
-        expect(card.content).to eq("DB Content")
+        expect(card.rule(:structure)).to eq('Formatted Content')
+        expect(card.content).to eq('DB Content')
       end
 
-      it "prefers a pattern virtual card to trash cards" do
-        Card.create!(name: "y+*right+*structure", content: "Formatted Content")
-        Card.create!(name: "a+y", content: "DB Content")
-        Card.fetch("a+y").delete!
+      it 'prefers a pattern virtual card to trash cards' do
+        Card.create!(name: 'y+*right+*structure', content: 'Formatted Content')
+        Card.create!(name: 'a+y', content: 'DB Content')
+        Card.fetch('a+y').delete!
 
-        card = Card.fetch("a+y")
+        card = Card.fetch('a+y')
         expect(card.virtual?).to be
-        expect(card.raw_content).to eq("Formatted Content")
+        expect(card.raw_content).to eq('Formatted Content')
       end
 
-      it "should recognize pattern overrides" do
-        #~~~ create right rule
-        tc=Card.create!(name: "y+*right+*structure", content: "Right Content")
-        card = Card.fetch("a+y")
+      it 'should recognize pattern overrides' do
+        # ~~~ create right rule
+        tc=Card.create!(name: 'y+*right+*structure', content: 'Right Content')
```

Comment:
Useless assignment to variable - `tc`.<br>Surrounding space missing for operator `=`.